micro_dl/cli/image_inference.py

```diff
@@ -229,33 +261,64 @@ def run_prediction(args, gpu_ids, gpu_mem_frac):
                     np.save(file_name, im_pred, allow_pickle=True)
                 else:
                     raise ValueError('Unsupported file extension')
+
+                # assuming target and predicted images are always 2D for now
+                # Load target
+                meta_idx = aux_utils.get_meta_idx(
+                    frames_meta,
+                    time_idx,
+                    target_channel,
```

Comment:
It looks like target_channel is only declared if using the --save_figs flag right now. Since it's being used no matter what, it probably makes sense to move it outside of that if statement.

Reply:
Yes! That makes more sense